contents/blog/measuring-feature-success.md

```diff
@@ -0,0 +1,133 @@
+featuredImageType: full
+category: Inside PostHog
+tags:
+ - Guides
+ - Feature flags
+---
+
+When it comes to new features, success can be tough to define. Although a business can succeed as a whole, what it means for a feature to succeed is less clear.
+
+At PostHog, we care about enabling teams to understand and measure feature success (we have [an entire team dedicated to it](/handbook/small-teams/feature-success)). As part of helping you do this, this post details what feature success is, and how we measure it at PostHog. We go through the real launch of a new feature (sampling) and explain how we measured success along the way.
+
+## Defining feature success
+
+Measuring feature success starts after deciding to work on a feature. We decide to build a feature based on quantitative and qualitative feedback combined with our thoughts on what’s best. Once we decide to build a feature, we start to judge its success based on if it:
+
+1. Solves a user problem. Key to this is understanding and defining users’ problems, then figuring out ways to solve them that fit with the product and business. 
+2. Is actually used. No matter how good you think your feature is, if it doesn’t get used, it isn’t benefitting users.
+
+Feature success is the process of enabling, measuring, and evaluating both of these. It is planning for how to develop a successful feature, a system for measuring its ongoing usage, and a process for evaluating its success.
+
+## How we measure feature success at PostHog
+
+Measuring feature success at PostHog is a multi-step process that goes hand-in-hand with our release process. The multi-step approach to feature success ensures we are building the right solution and that users actually care and use the solution we are building.
+
+To help you connect this to reality, we will follow along with how one of our engineers, [Yakko](/handbook/company/team), recently built, measured, and evaluated [sampling](/docs/product-analytics/sampling).
+
+> **What is sampling?** The sampling feature speeds up queries by analyzing a portion of the data and extrapolating the results.
+
+![Sampling in action](../images/blog/measuring-feature-success/sampling.gif)
+
+### 0. Deciding to build a feature
+
+A feature needs to exist for us to measure its success. Deciding to build a feature comes before everything in the feature success process.
+
+From the thinking and decision about building a feature comes the criteria for evaluating its success. 
+
+Yakko decided to work on sampling because:
+1. Slow queries were causing frustrations
+2. Sampling felt like an intuitive solution that could be implemented quickly
+3. Manually benchmarking the impact of 10% sampling on large customers' queries confirmed 3-10x improvements in speeds with only a 1-2% loss in accuracy.
+
+This showed that sampling was technically feasible and it was worth doing now.
+
+### 1. Build it and make sure it isn’t broken
```

Comment:
This is a slightly negative framing... maybe something more like "Build an MVP and test it's working as intended"